cognitive_mapping_and_planning/tfcode/vision_baseline_lstm.py

```diff
@@ -196,6 +197,7 @@ def combine_setup(name, combine_type, embed_img, embed_goal, num_img_neuorons=No
       out = embed_goal
     else:
       logging.fatal('Undefined combine_type: %s', combine_type)
+      raise ValueError('Undefined combine_type: %s', combine_type)
```

Comment:
Although `logging.fatal` is not documented it seems to be equivalent to `logging.critical` (see [code](https://github.com/python/cpython/blob/2.7/Lib/logging/__init__.py)) so I wouldn't expect that line to crash?

However I agree that simply raising an exception seems better - and if we wanted to log it that should be done by whatever catches the exception (`logging ` has some convenience method for that as well). Will update!